include/robot_localization/ros_filter.h

```diff
@@ -549,8 +558,35 @@ template<class T> class RosFilter
     //! @brief tf frame name that is the parent frame of the transform that this node will calculate and broadcast.
     //!
     std::string worldFrameId_;
-};
 
+    //! @brief Publisher to publish the filtered pose
+    //!
+    ros::Publisher positionPub_;
+
+    //! @brief map to odom transform
+    //!
+    tf2::Transform mapOdomTrans_;
+
+    //! @brief odom to BaseLink transform
+    //!
+    tf2::Transform odomBaseLinkTrans_;
+
+    //! @brief tf message
+    //!
+    geometry_msgs::TransformStamped mapOdomTransMsg_;
+
+    //! @brief tf broadcaster
+    //!
+    tf2_ros::TransformBroadcaster worldTransformBroadcaster_;
+
+    //! @brief mutex for the mesurement queue
+    //!
+    boost::mutex measurementQueueMutex_;
+
+    //! @brief condition variable for the measurement queue
+    //!
+    boost::condition_variable measurementQueueEmpty_;
```

Comment:
Again, not critical, but it appears that this mutex is actually used for notifying the integrateMeasurements method that the queue is *not* empty. For clarity, would it make sense to call it something like measurementsReady_?